internal/handlers/pg/pgdb/database_metadata.go

```diff
@@ -218,20 +225,24 @@ func (m *metadata) remove(ctx context.Context) error {
 	return lazyerrors.Error(err)
 }
 
-// formatCollectionName returns collection name in form <shortened_name>_<name_hash>.
-// Changing this logic will break compatibility with existing databases.
-func formatCollectionName(name string) string {
+// transform returns name in form <shortened_name>_<name_hash>.
+// It replaces special characters with `_`.
+//
+// Deprecated: this function usage is allowed for collection metadata creation only.
+func transform(name string) string {
```

Comment:
Independently of "mangling"/"transform"/"format" discussion, that name is too short for a package of a size of pgdb. Transform what? Some string to some string?

`transformCollectionName` would be better. Another option could be `collectionNameToTableName`, but that's maybe a bit long

Reply:
perhaps `collectionNameToTableName` is good for this one, even it's a bit long, it specifies the user case and we don't want anything else to be using this/